### PR TITLE
Branch out a variant of default config that does not scrape processes metrics.

### PR DESCRIPTION
**Pull request by qingling128**
This way we can refer to this file in public documentation for easily turning off processes metrics.

### CODE REVIEW

File: collectd-no-processes.conf [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# This file is a variant of the default collectd.conf except processes metrics are not collected.
```
Comment (igorpeshansky) [inline collectd-no-processes.conf:1]: Nit: s/`processes metrics`/`process metrics`/

Comment (igorpeshansky) [inline collectd-no-processes.conf:1]: My worry is that we would have to keep these two files in sync as we make changes, which is error-prone… Not much we can do about it, though.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]

File: collectd-no-processes.conf (hunk 1 above)
Comment (qingling128) [reply collectd-no-processes.conf]: Done

Comment (qingling128) [reply collectd-no-processes.conf]: Yeah. Another way to think about this is that if we do not do it, a number of customers would need to do it. So...

**Review by qingling128 — COMMENTED** [review]
Updated

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

File: collectd-no-process-metrics.conf [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# This file is a variant of the default collectd.conf with no process metrics.
```
Comment (igorpeshansky) [inline collectd-no-process-metrics.conf:1]: Actually, I'd suggest `with process metrics disabled`, to imply that the default behavior is to collect them.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
Sorry, one more thought.

File: collectd-no-process-metrics.conf (hunk 2 above)
Comment (qingling128) [reply collectd-no-process-metrics.conf]: Updated.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**